src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/OverseasEntitySubmissionDto.java

```diff
@@ -8,16 +8,19 @@
 
 public class OverseasEntitySubmissionDto {
 
+    public static final String ENTITY_FIELD = "entity";
+    public static final String BENEFICIAL_OWNERS_INDIVIDUAL_FIELD = "beneficial_owners_individual";
+
     @JsonProperty("presenter")
     private PresenterDto presenter;
 
-    @JsonProperty("entity")
```

Comment:
what : entity_field
why: concern: will need to amend the stack to get it to agree with the new name
where: just a question really: is there any way of qualifying it when it goes to the filing service?

Reply:
@mattch1 no need to 'amend the stack' (do you mean spec, web and other model classes?) because name hasn't actually changed. It's still "entity" but now just referencing a constant.

Does that answer your question?